app/gimlet/rev-a.toml

```diff
@@ -545,6 +545,153 @@ pmbus = { rails = [ "V12_SYS_A2" ] }
 sensors = { temperature = 1, power = 1, voltage = 1, current = 1 }
 refdes = "U431"
 
+################################################################################
+# DIMM slots
+[[config.i2c.devices]]
```

Comment:
I think on these we want to indicate that `removable = true` -- which should allow `humility validate` to do the right thing when they're gone.